File: python/svix/models/list_response_endpoint_message_out.py
# this file is @generated
import typing as t

from .common import BaseModel
from .endpoint_message_out import EndpointMessageOut


class ListResponseEndpointMessageOut(BaseModel):
    data: t.List[EndpointMessageOut]

    done: bool

    iterator: t.Optional[str]

    prev_iterator: t.Optional[str] = None

File: csharp/Svix/Models/ListResponseEndpointMessageOut.cs
// this file is @generated
using Newtonsoft.Json;

namespace Svix.Models
{
    public class ListResponseEndpointMessageOut : BaseModel
    {
        [JsonProperty("data")]
        public required List<EndpointMessageOut> Data { get; set; }

        [JsonProperty("done")]
        public required bool Done { get; set; }

        [JsonProperty("iterator")]
        public string? Iterator { get; set; } = null;

        [JsonProperty("prevIterator")]
        public string? PrevIterator { get; set; } = null;
    }
}
